### PR TITLE
Add public method in BluetoothPeripheralManager  public boolean notif…

### PR DESCRIPTION
**Pull request by aknabi**
Add public method in BluetoothPeripheralManager notifyCharacteristicChanged(byte[] value, BluetoothCentral bluetoothCentral,BluetoothGattCharacteristic characteristic),

### CODE REVIEW

File: blessed/src/main/java/com/welie/blessed/BluetoothPeripheralManager.java [hunk 1]
```diff
@@ -607,6 +627,15 @@ private void cancelConnection(@NotNull final BluetoothDevice bluetoothDevice) {
         return bluetoothManager.getConnectedDevices(BluetoothGattServer.GATT);
     }
 
+    private BluetoothDevice getDeviceForCentral(@NotNull final BluetoothCentral bluetoothCentral) {
```
Comment (weliem) [inline blessed/src/main/java/com/welie/blessed/BluetoothPeripheralManager.java:630]: This entire method is unnecessary. Can we replaced by:

bluetoothAdapter.getRemoteDevice(bluetoothCentral.getAddress())

File: blessed/src/main/java/com/welie/blessed/BluetoothPeripheralManager.java [hunk 2]
```diff
@@ -562,6 +562,26 @@ public boolean notifyCharacteristicChanged(@NotNull final byte[] value, @NotNull
...
+     *
+     * <p>A notification or indication is sent to the remote centrals to signal
+     * that the characteristic has been updated. If the connected central is not
+     * found then false is return
+     *
+     * @param characteristic the characteristic for which to send a notification
+     * @return true if the central exists and the operation was enqueued, otherwise false
+     */
+    public boolean notifyCharacteristicChanged(@NotNull final byte[] value, @NotNull final BluetoothCentral bluetoothCentral, @NotNull final BluetoothGattCharacteristic characteristic){
+        BluetoothDevice bluetoothDevice = getDeviceForCentral(bluetoothCentral);
```
Comment (weliem) [inline blessed/src/main/java/com/welie/blessed/BluetoothPeripheralManager.java:577]: Please add checks to confirm that parameters are not null:

        Objects.requireNonNull(bluetoothCentral, CENTRAL_IS_NULL);
        Objects.requireNonNull(value, CHARACTERISTIC_VALUE_IS_NULL);
        Objects.requireNonNull(characteristic, CHARACTERISTIC_IS_NULL);

**Review by weliem — COMMENTED** [review]
Added some minor comments...

File: blessed/src/main/java/com/welie/blessed/BluetoothPeripheralManager.java (hunk 1 above)
Comment (aknabi) [reply blessed/src/main/java/com/welie/blessed/BluetoothPeripheralManager.java]: Removed method and changed usage to your above code snippet

File: blessed/src/main/java/com/welie/blessed/BluetoothPeripheralManager.java (hunk 2 above)
Comment (aknabi) [reply blessed/src/main/java/com/welie/blessed/BluetoothPeripheralManager.java]: Added to top of method

**Review by weliem — APPROVED** [review]

**Pull request merged**